calico-policies/private-network-isolation/calico-v3/us-south/allow-private-service-endpoint.yaml

```diff
@@ -54,17 +54,21 @@ spec:
       - 166.9.17.37/32
       - 166.9.17.39/32
       - 166.9.48.171/32
+      - 166.9.48.175/32
       - 166.9.48.124/32
       - 166.9.48.50/32
       - 166.9.48.76/32
       - 166.9.51.16/32
       - 166.9.51.54/32
       - 166.9.51.74/32
       - 166.9.51.106/32
+      - 166.9.51.104/32
       - 166.9.58.11/32
       - 166.9.58.16/32
       - 166.9.58.65/32
       - 166.9.58.104/32
+      - 166.9.58.170/32
+      - 166.9.59.7/32
```

Comment:
I think this one, 166.9.59.7 still needs to be removed (it was removed from the other 3 places)

Reply:
🤦